### DISCUSSION TITLE
Default to "Request power down" when quitting app or closing a VM window?

### DISCUSSION THREAD

**Discussion by ireuben** [utmapp/UTM#5480] [Ideas]
It seems like it would be more friendly behavior if the default was to "Request power down" for a VM if you are quitting the app or closing a VM window, so it would shut down gracefully and not mess anything up. If that doesn't work, then the user could decide to force a shut down or kill. 

As a bonus, could have a timeout (30 seconds?) which then pops up a dialog saying VM doesn't seem to be responding to the request to shutdown, so would you like to force it and possibly lose data?

**Comment by osy** [comment]
The problem is that “request” will often pop up a dialog in the VM where you have to pick “shut down”. In that case, you might as well have shut down the vm from the guest before closing it anyways.

**Reply by ireuben** [reply]
In my case, I'm using Windows 11 and a command-line only linux, and they both shut down automatically after a "Request power down". 

Could also see this as a pref ("When closing a running VM: [Request power down]", with the same three menu options as the power icon on VM window). But does seem like a default of "Request power down" would be ideal both for usability (it just works with some VMs) and data safety (won't mess up your VM's drive, and worst case you have to click Shutdown on the prompt inside VM).

**Reply by osy** [reply]
Feel free to make this a feature request in a new issue
